src/RA_Dlg_Memory.cpp

```diff
@@ -1079,7 +1081,7 @@ INT_PTR Dlg_Memory::MemoryProc(HWND hDlg, UINT nMsg, WPARAM wParam, LPARAM lPara
                         {
                             ra::tstring buffer = nativeBuffer;
                             //	Read hex or dec
-                            if (buffer[0] == '0' && buffer[1] == 'x')
+                            if (buffer.substr(0, 2) == "0x")
```

Comment:
prefer not creating a temporary string for the comparison. recommend using `ra::StringStartsWith`, which expands to `buffer.compare(0, 2, "0x") == 0)`